main/index.js

```diff
@@ -0,0 +1,18 @@
+import display from '../modules/displayfunction.js';
```

Comment:
- [ ] Kindly refactor your functions in all modules to arrow functions as per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/books/books_with_es6.md#:~:text=Refactor%20your%20functions%20to%20arrow%20functions.).